genesyscloud/resource_genesyscloud_architect_emergencygroup.go

```diff
@@ -0,0 +1,308 @@
+				Computed:    true,
+			},
+			"description": {
+				Description: "Description of the emergency group.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"enabled": {
+				Description: "The state of the emergency group. Defaults to false/inactive.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+			},
+			"emergency_call_flows": {
+				Description: "The emergency call flows for this emergency group.",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"emergency_flow_id": {
+							Description: "The ID of the connected call flow.",
+							Type:        schema.TypeString,
+							Required:    true,
+						},
+						"ivr_ids": {
+							Description: "The IDs of the connected IVRs.",
+							Type:        schema.TypeSet,
+							Required:    true,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func createEmergencyGroup(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	name := d.Get("name").(string)
+	description := d.Get("description").(string)
+	divisionId := d.Get("division_id").(string)
+	enabled := d.Get("enabled").(bool)
+
+	sdkConfig := meta.(*providerMeta).ClientConfig
+	architectAPI := platformclientv2.NewArchitectApiWithConfig(sdkConfig)
+
+	emergencyGroup := platformclientv2.Emergencygroup{
+		Name:               &name,
+		Enabled:            &enabled,
+		EmergencyCallFlows: buildSdkEmergencyGroupCallFlows(d),
+	}
+
+	// Optional attributes
+	if description != "" {
+		emergencyGroup.Description = &description
+	}
+
+	if divisionId != "" {
+		emergencyGroup.Division = &platformclientv2.Division{Id: &divisionId}
+	}
+
+	log.Printf("Creating emergency group %s", name)
+	eGroup, _, err := architectAPI.PostArchitectEmergencygroups(emergencyGroup)
+	if err != nil {
+		return diag.Errorf("Failed to create emergency group %s: %s", name, err)
+	}
+
+	d.SetId(*eGroup.Id)
+
+	log.Printf("Created emergency group %s %s", name, *eGroup.Id)
+
+	return readEmergencyGroup(ctx, d, meta)
+}
+
+func readEmergencyGroup(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	sdkConfig := meta.(*providerMeta).ClientConfig
+	architectApi := platformclientv2.NewArchitectApiWithConfig(sdkConfig)
+
+	log.Printf("Reading emergency group %s", d.Id())
+	return withRetriesForRead(ctx, 30*time.Second, d, func() *resource.RetryError {
+		emergencyGroup, resp, getErr := architectApi.GetArchitectEmergencygroup(d.Id())
+		if getErr != nil {
+			if isStatus404(resp) {
+				return resource.RetryableError(fmt.Errorf("Failed to read emergency group %s: %s", d.Id(), getErr))
+			}
+			return resource.NonRetryableError(fmt.Errorf("Failed to read emergency group %s: %s", d.Id(), getErr))
+		}
+
+		if emergencyGroup.State != nil && *emergencyGroup.State == "deleted" {
+			d.SetId("")
+			return nil
+		}
+
```

Comment:
Could you rebase off the latest master and add the following line here: https://github.com/MyPureCloud/terraform-provider-genesyscloud/pull/237/files#diff-66a437ba0ed6d576f4cba11112118036be5d7122cd78f9ceda920f9d51bff448R227 replacing `resourceProcessAutomationTrigger` with `resourceArchitectEmergencyGroup`